projects/angular/src/forms/input/input.spec.ts

```diff
@@ -39,5 +42,48 @@ export default function (): void {
     ControlStandaloneSpec(StandaloneUseTest);
     TemplateDrivenSpec(ClrInputContainer, ClrInput, TemplateDrivenTest, 'clr-input');
     ReactiveSpec(ClrInputContainer, ClrInput, ReactiveTest, 'clr-input');
+    inputSpec('tempate-driven', ClrInputContainer, ClrInput, TemplateDrivenTest);
+    inputSpec('reactive', ClrInputContainer, ClrInput, ReactiveTest);
+  });
+}
+
+function inputSpec(description, testContainer, testControl, testComponent) {
```

Comment:
Though these tests seem to belong in `control.spec.ts` together with the other template driven and reactive tests, I have to make them input-specific, as many controls, like checkboxes, selects etc, don't emit `valueChanges`.